songbank-backend/backend.js

```diff
@@ -115,6 +115,36 @@ app.get('/auth/search/:id', (req, res) => {
     });
 })
 
+app.get('/auth/search/:id/:artist', (req, res) => {
+    //songTitle = '';
+    const id = req.params['id'];
+    const artist = req.params['artist'];
+    console.log(access_token);
+    console.log(id);
+    var searchOptions = {
+        url: `https://api.spotify.com/v1/search?q=${id}+${artist}&type=track%2Cartist&limit=5`,
+        headers: {
+            'Accept' : 'application/json',
+            'Content-Type' : 'application/json',
+            'Authorization' :  `Bearer ${access_token}`
+        },
+    };

```

Comment:
Really cool to see that we got searching for songs working! Well written too and easy to read/understand, nice job!

Reply:
Sounds good! thank you!